.landscaper/resources.yaml

```
@@ -0,0 +1,19 @@
+---
+  type: landscaper.gardener.cloud/blueprint
+  name: os-gardenlinux-controller-registration
+  relation: local
+  input:
+    type: "dir"
+    path: "./blueprint"
+    compress: true
+    mediaType: "application/vnd.gardener.landscaper.blueprint.v1+tar+gzip"
```

```suggestion
    mediaType: "application/vnd.gardener.landscaper.blueprint.layer.v1.tar+gzip"
```
update to the newer oci spec compliant media types